reco_utils/evaluation/python_evaluation.py

```diff
@@ -648,17 +650,21 @@ def get_top_k_items(
         customerID-itemID-rating)
         col_user (str): column name for user
         col_rating (str): column name for rating
-        k (int): number of items for each user
+        k (int or None): number of items for each user; None means that the input has already been
```

Comment:
Is this something related to `relevancy_method=None`?

Reply:
added comments